### ISSUE TITLE
App creased am using Xcode 15 and ios 17.0 am used

### ISSUE THREAD

**Issue by vinodlee** [spencerccf/app_settings#195]
App creased am using Xcode 15 and ios 17.0 am used need help with this.

**Comment by navaronbracke** [comment]
So you are using XCode 15, with iOS 17.0, right? What exactly is crashing for you?

**Comment by navaronbracke** [comment]
As the stack trace indicates a crash in `Dispatch queue: com.apple.device-scene-loader`, I have reported this crash to Apple.

**Comment by vinodlee** [comment]
Hi @navaronbracke Any update about this app crash issue?

**Comment by navaronbracke** [comment]
There have been two patches to iOS 17, the latest of which is iOS 17.0.2

I will test if I can reproduce the crash on that version.

**Comment by vinodlee** [comment]
If you find any solution for this please let me know @navaronbracke

**Comment by navaronbracke** [comment]
@vinodlee My first guess would be that an update to iOS 17 fixes this bug. In that case, no further action is needed for the plugin.

However, if Apple decides that the usage of this API needs to change to fix the bug, (i.e. adding an entitlement or a new parameter) then I will look into adjusting the code in the `app_settings` plugin.

**Comment by navaronbracke** [comment]
@vinodlee Can confirm, the crash is fixed in iOS 17.0.1 for the iOS Simulator.

Had to upgrade to MacOS Sonoma and XCode 15.0.1 

```
[✓] Flutter (Channel stable, 3.13.7, on macOS 14.0 23A344 darwin-x64, locale
    en-BE)
    • Flutter version 3.13.7 on channel stable at
      /Users/navaronbracke/Documents/flutter
    • Upstream repository git@github.com:navaronbracke/flutter.git
    • FLUTTER_GIT_URL = git@github.com:navaronbracke/flutter.git
    • Framework revision 2f708eb839 (12 days ago), 2023-10-09 09:58:08 -0500
    • Engine revision a794cf2681
    • Dart version 3.1.3
    • DevTools version 2.25.0

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/navaronbracke/Library/Android/sdk
    • Platform android-34, build-tools 34.0.0
    • ANDROID_HOME = /Users/navaronbracke/Library/Android/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      17.0.6+0-17.0.6b829.9-10027231)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 15.0.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 15A507
    • CocoaPods version 1.13.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2022.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build
      17.0.6+0-17.0.6b829.9-10027231)

[✓] VS Code (version 1.83.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.74.0

[✓] Connected device (3 available)            
    • iPhone 15 (mobile) • AA4D047E-6334-4F14-9E90-1EFBCF713DC7 • ios
      • com.apple.CoreSimulator.SimRuntime.iOS-17-0 (simulator)
    • macOS (desktop)    • macos                                • darwin-x64
      • macOS 14.0 23A344 darwin-x64
    • Chrome (web)       • chrome                               • web-javascript
      • Google Chrome 118.0.5993.88

[✓] Network resources
    • All expected network resources are available.

• No issues found!
```
